src/ResourceManagement/HDInsight/HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,4 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("0.9.0.0*")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.0.2.0")]
```

Comment:
you should separate out the file version, which will be in different updating cadence with assembly version such as you are just tweaking the build number and want to keep the assembly version so not to break your client app